L2-counter/scripts/increment-counter.ts

```diff
@@ -0,0 +1,75 @@
+import { BigNumber, Contract, ethers, Wallet } from 'ethers';
+import { Provider, utils } from 'zksync-web3';
+
+const GOVERNANCE_ABI = require('./governance.json');
+const GOVERNANCE_ADDRESS = '<GOVERNANCE-CONTRACT-ADDRESS>';
+const COUNTER_ABI = require('./counter.json');
+const COUNTER_ADDRESS = '<COUNTER-CONTRACT-ADDRESS>';
+
+async function main() {
+  // Ethereum L1 provider
+  const l1Provider = ethers.providers.getDefaultProvider('goerli');
+
+  // Governor wallet
+  const wallet = new Wallet('<YOUR-PRIVATE-KEY>', l1Provider);
+
+  const govcontract = new Contract(GOVERNANCE_ADDRESS, GOVERNANCE_ABI, wallet);
+
+  // Getting the current address of the zkSync L1 bridge
+  const l2Provider = new Provider('https://zksync2-testnet.zksync.dev');
+  const zkSyncAddress = await l2Provider.getMainContractAddress();
+  // Getting the `Contract` object of the zkSync bridge
+  const zkSyncContract = new Contract(
+    zkSyncAddress,
+    utils.ZKSYNC_MAIN_ABI,
+    wallet
+  );
+
+  // Encoding the tx data the same way it is done on Ethereum.
+  const counterInterface = new ethers.utils.Interface(COUNTER_ABI);
+  const data = counterInterface.encodeFunctionData('increment', []);
+
+  // The price of the L1 transaction requests depends on the gas price used in the call
+  const gasPrice = await l1Provider.getGasPrice();
+
+  // Here we define the constant for ergs limit.
```

Comment:
why all of a sudden periods at the end of the comment?

Reply:
fixed in c524d3a1470771290f717b27c219f20ecab10e72